src/renderer/src/components/icon-button/IconButton.tsx

```diff
@@ -1,9 +1,13 @@
-import "./styles.css";
 import { Component, JSX } from "solid-js";
 
 type Props = JSX.IntrinsicElements["button"];
 const IconButton: Component<Props> = (props) => {
-  return <button {...props} class={`icon-button ${props.class}`} />;
+  return (
+    <button
+      {...props}
+      class={`-m-2 flex h-6 w-6 items-center justify-center rounded-full p-2 text-2xl hover:bg-surface focus:bg-surface ${props.class}`}
```

Comment:
You can use `cn` here.

Reply:
I think I'm just gonna remove icon button entirely

Reply:
Add is a a variant of the button component instead?

Reply:
Yeah